docs: revision (#9)
spectrocloud/docs-assignments: technical-writer/instructions.md

- You can use Gitlab, GitHub, Bitbucket, or any other versioning control system to host your deliverable.
- Assume the content would reside in a formal documentation site.
- Users of all skill level will read the content.

> - This is not a [tutorial](https://docs.spectrocloud.com/clusters/public-cloud/deploy-k8s-cluster), nor a [procedural document](https://docs.spectrocloud.com/clusters/cluster-management/remove-clusters) (Step 1 ... Step N). The content is intended to be a reference page (also known as a concept page) that both beginners and experts can use to find information about a topic. The content should provide enough information to help the reader understand the concept and how to use the various commands.

Reviewer: ```suggestion
- This is not a [tutorial](https://docs.spectrocloud.com/clusters/public-cloud/deploy-k8s-cluster), nor a [procedural document](https://docs.spectrocloud.com/clusters/cluster-management/remove-clusters) (Step 1 ... Step N). The content is intended to be a reference page for beginners and experts to find information about a topic. The content should provide enough information to help the reader understand the concept and how to use the various commands.
```

Reviewer: I suggest removing the parenthetical alternate name for _reference_ because it might confuse writers familiar with DITA topic types (concept, task, reference, and principle) - or they might wonder if it's a trick. _Reference_ is clear enough.
